cdap-common/src/test/java/co/cask/cdap/common/security/UGIProviderTest.java

```diff
@@ -175,7 +175,7 @@ public InetSocketAddress getSocketAddress() {
 
       // Create Alice UGI
       RemoteUGIProvider ugiProvider = new RemoteUGIProvider(cConf, discoveryService, locationFactory);
-      ImpersonationInfo aliceInfo = new ImpersonationInfo(getPrincipal("alice"), keytabFile.toURI().toString());
+      ImpersonationInfo aliceInfo = new ImpersonationInfo(getPrincipal("alice"), keytabFile.getAbsolutePath());
```

Comment:
Should keep this one to make sure both path (`/some/keytab/path`) and URI (`file:///some/keytab/path`) works.